src/main/java/com/lob/protocol/response/AbstractResponseList.java

```diff
@@ -8,7 +8,7 @@
 
 import static com.lob.Util.defensiveCopy;
 
-public abstract class AbstractResponseList<T> implements Iterable<T> {
+public abstract class AbstractResponseList<T> extends AbstractResponse implements Iterable<T> {
```

Comment:
I see. I read the class name as a list of responses, but I now realize that's silly as there's only one response...